Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp

```diff
@@ -0,0 +1,56 @@
+//====================================================================
+//  Detector description implementation of the Drift Chamber
+//--------------------------------------------------------------------
+//
+//  Author: Tao Lin
+//
+//====================================================================
+
+#include "DD4hep/DetFactoryHelper.h"
+#include "XML/Layering.h"
+#include "XML/Utilities.h"
+#include "DDRec/DetectorData.h"
+#include "DDSegmentation/Segmentation.h"
+
+#define MYDEBUG(x) std::cout << __FILE__ << ":" << __LINE__ << ": " << x << std::endl;
+#define MYDEBUGVAL(x) std::cout << __FILE__ << ":" << __LINE__ << ": " << #x << ": " << x << std::endl;
+
+static dd4hep::Ref_t create_detector(dd4hep::Detector& theDetector,
+                                     xml_h e,
+                                     dd4hep::SensitiveDetector sens) {
+    xml_det_t x_det = e;
+
+    std::string det_name = x_det.nameStr();
+    std::string det_type = x_det.typeStr();
+
+    dd4hep::DetElement sdet(det_name, x_det.id());
+
+    dd4hep::Volume motherVol = theDetector.pickMotherVolume(sdet);
+
+    dd4hep::Material det_mat(theDetector.material("Air"));
+    
+    dd4hep::Tube det_solid(23.5, 171.6, 222.5);
```

Comment:
Shall we make the numbers based on a unified unit system in the CEPCSW?

Reply:
The dimensions are moved to the xml file. See commit 2d3a664c9e47106d56d127f5708a066ec17a7b1f